src/authorized_keys/serializers.py

```diff
@@ -6,14 +6,20 @@
 class ReverseServerAuthorizedKeysSerializer(serializers.ModelSerializer):
     class Meta:
         model = ReverseServerAuthorizedKeys
-        fields = ['id', 'hostname', 'key', 'reverse_port', 'description', 'created_at', 'updated_at']
+        fields = '__all__'
```

Comment:
I think it's still a better way to list all the field, rather than just use `__all__`.

Reply:
In my sight, the fields of database table cannot be changed constantly.
So, we can easily know them without showing all fields. 🥳